feature_engine/variable_manipulation.py

```diff
@@ -283,3 +283,34 @@ def _find_all_variables(
         X[variables]
 
     return variables
+
+
+def _filter_out_variables_not_in_dataframe(X, variable_list):
```

Comment:
I would call variable_list just `variables` so we have the same name across all functions and transformers. We are always talking about the same thing: the variable list entered by the user.

Reply:
Done